CONTRIBUTING.md

```diff
@@ -1,102 +1,51 @@
-
-Further paragraphs come after blank lines.
-
- - Bullet points are okay, too
-
- - Typically a hyphen or asterisk is used for the bullet, preceded
-   by a single space, with blank lines in between, but conventions
-   vary here
-
-If you use an issue tracker, put references to them at the bottom,
-like this:
-
-Resolves: #123
-See also: #456, #789
-```
-
-### 5. Rebasing pull requests
-When different people are working on the Arduino project simultaneously, pull requests can go stale quickly. A "stale" pull request is one that is no longer up to date with the latest merges in the project. It needs to be updated before it can be merged.
-
-Most often pull requests become stale when merge conflicts occur. This happens when two pull requests both modify similar lines in the same file and one gets merged, the unmerged request will now have a merge conflict and needs updating.
-
-When your pull request is stale, you will need to rebase your branch on the current master branch before you can merge it without conflicts.
-
-More information about rebasing can be found at the repository of [edX](https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request).
-
-### 6. Merged!
-When your pull request is merged please update the documentation if the changes require it:
-
-- [ ] Edit appropiate [Wiki pages](https://github.com/arduino/Arduino/wiki/_pages)
-- [ ] Submit an [issue report](https://github.com/arduino/Arduino/issues/new) requesting changes to the [arduino.cc reference pages](https://www.arduino.cc/en/Reference/HomePage)
+## Contributing Rules
+Thanks for your interest in contributing to this free open source project! Please take the time to read this before submitting an issue report or pull request.
+
+### Issues
+- Do you need help or have a question about using Arduino? Support requests should be made to the appropriate section of the [Arduino forum](http://forum.arduino.cc) rather than an issue report. **Issue reports are to be used to report bugs or make feature requests only.**
+- Check if your issue has already been resolved in the [hourly build](http://www.arduino.cc/en/Main/Software#hourly).
+- Submit issue reports to the correct repository:
+
+Issue subject | Report at
+---|---
+Arduino IDE, arduino.cc (but not the Arduino Playground), Library Manager additions | [arduino/Arduino](https://github.com/arduino/Arduino)
+[Arduino Playground](http://playground.arduino.cc/) | This is a publicly editable wiki. Please either make the edit yourself or create a post on the [forum](http://forum.arduino.cc/index.php?board=24.0).
+Arduino Forum issues | [arduino/forum-issues](https://github.com/arduino/forum-issues)
+Arduino libraries | [arduino-libraries](https://github.com/arduino-libraries)
+arduino-builder | [arduino/arduino-builder](https://github.com/arduino/arduino-builder)
+[Arduino Web Editor](https://create.arduino.cc/editor) | [arduino/arduino-create-agent](https://github.com/arduino/arduino-create-agent)
```

Comment:
I see there is an empty repository: https://github.com/arduino/arduino-create but there is no explanation of the purpose. I was told there is a private issue tracker and some plans to create a public one but I haven't heard anything about it since. I realize that the arduino-create-agent repository is not the ideal place for general Arduino Web Editor issue reports unrelated to the agent.